ingestion/operators/docker/Dockerfile

```diff
@@ -39,7 +39,19 @@ RUN apt-get update && \
     cd /tmp && git clone https://github.com/edenhill/librdkafka.git && \
     cd librdkafka && git checkout tags/v1.9.0 && \
     ./configure && make && make install && \
-    cd ../ && rm -rf librdkafka
+    cd ../ && rm -rf librdkafka \
+
+# Oracle instant client for thick mode
+# https://python-oracledb.readthedocs.io/en/latest/user_guide/initialization.html#enablingthick
+RUN apt-get update; \
+  apt-get install libaio1 alien unzip -y;
```

Comment:
Instead of separately running another Docker `RUN` Instruction, let's include these in the already available `RUN` instruction which installs apt dependencies!
![image](https://user-images.githubusercontent.com/15995028/213214640-dd0a4306-afa6-4e9f-865a-94688c8ad5c0.png)